src/EventListener/Dca/FieldOptionsCallbackListener.php

```diff
@@ -38,6 +44,11 @@ public function __invoke(?DataContainer $dc)
                 }
 
                 break;
+
+            case'tl_content':
+                if ($this->dcaUtil) {
+                    return $this->dcaUtil->getFields($dc->table, []);
```

Comment:
Damit bin ich noch nicht zufrieden. Du hast es zwar als optionale Abhängigkeit definiert, aber einige Syndication Types funktionieren ohne die Zuweisung nicht. Daher muss das Utils-Bundle installiert sein, um das Element korrekt lauffähig zu haben. Allerdings würde ich hier ungern das Utils-Bundle als abhängigkeit reinnehnmen, habe es extra rausgelassen :P
BTW.: Eventuell sollte man die Daten gar nicht von tl_content auslesen, sondern von tl_page. Dann bräuchtest du auch kein Page-Title-Feld. Dann müsstest du stattdessen ein einfaches Mapping in das Content Element einbauen.